formula/src/main/java/com/instacart/formula/Action.kt

```diff
@@ -190,12 +195,11 @@ interface Action<Event> {
      * to clean up such as remove subscriptions to RxJava observables, Kotlin Flows, event buses,
      * etc.
      *
-     * @param send Use this listener to send events back to [Formula].
-     *             Note: you need to call this on the main thread.
+     * @param emitter Use emitter to send events back to [Formula].
```

Comment:
No more main thread requirement?

Reply:
Nope, it was removed when I added multi-thread support over a year ago.